Source/SeaInk.Endpoints/Server/Controllers/MentorController.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using System.Linq;
+using Infrastructure.Database;
+using Microsoft.AspNetCore.Mvc;
+using MoreLinq;
+using SeaInk.Core.Entities;
+using SeaInk.Endpoints.Shared.Dto;
+
+
+namespace SeaInk.Endpoints.Server.Controllers
+{
+    [Route("[controller]")]
+    public class MentorController : Controller
+    {
+        private readonly DatabaseContext _databaseContext;
+
+        public MentorController(DatabaseContext databaseContext)
+        {
+            _databaseContext = databaseContext;
+        }
+
+
+        [HttpGet("{mentorId}/subjects")]
+        public List<SubjectDto> GetSubjectsList(int mentorId)
+        {
+            Mentor mentor = _databaseContext.Mentors.Find(mentorId);
+
+            if (mentor is null)
+                return new List<SubjectDto>();
+
+            List<Division> mentorDivisions = mentor.Divisions;
+            return mentorDivisions.Select(x => x.Subject.ToDto()).ToList();
+        }
+
+        [HttpGet("{mentorId}/subject/{subjectId}/groups")]
+        public List<StudyGroupDto> GetGroupsList(int mentorId, int subjectId)
+        {
+            Mentor mentor = _databaseContext.Mentors.Find(mentorId);
+
+            if (mentor is null)
+                return new List<StudyGroupDto>();
+
+            List<Division> mentorDivisions = mentor.Divisions;
+            return mentorDivisions
```

Comment:
```suggestion
            return mentor.Divisions
```
Тут вроде как лист этот не особо полезный